src/melee/ft/chara/ftSeak/ftSk_SpecialLw.c

```diff
@@ -13,12 +14,12 @@
 static float const ftSk_Init_804D9698 = 0.4000000059604645F;
 static float const ftSk_Init_804D969C = 0.003000000026077032F;
 static float const ftSk_Init_804D96A0 = 1.0471975803375244F;
-static float const ftSk_Init_804D96A4 = 0.0F;
-static double const ftSk_Init_804D96A8 = 0.0078125;
+static f32 const ftSk_Init_804D96A4 = 0.0f;
+static f32 const ftSk_Init_804D96A8 = 0.0078125f;
 
-/// #fn_80114034
+void fn_80114034(HSD_GObj* gobj) {}
 
-/// #fn_801140B0
+void fn_801140B0(void) {}
```

Comment:
ah sorry, I guess I led you astray here - looks like this is assigned to `accessory4_cb` so it needs to be defined as `void fn_801140B0(HSD_GObj* gobj)`, the gobj argument is just unused